src/components/atoms/Button/index.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import './style.scss';
 
-const Button = ({ className, arrowDirection }) => (
-  <button type="button" className={className}>
-    <div className={arrowDirection} />
+const Button = ({ className, arrowDirection, onClick }) => (
+  <button type="button" className={className} onClick={onClick}>
+    <div className={className === 'button--big' ? `arrow--${arrowDirection} arrow--big` : `arrow--${arrowDirection}`} />
```

Comment:
dont crewate logic based on classname, if i want to use your component with different or additional class name, then I will bereak this element. Please create logic based on arrowDirection property

Reply:
Then i need to add button size property, because arrow size depends on the size of the button.